CMakeLists.txt

```diff
@@ -17,7 +17,7 @@
 # Mandatory call to project
 project(opm-grid C CXX)
 
-cmake_minimum_required (VERSION 2.8)
+cmake_minimum_required (VERSION 3.0)
```

Comment:
This should probably say `3.10` instead.

Reply:
Thanks for pointing this out. Fixed,